components/kyma-environment-broker/internal/process/upgrade_kyma/connectivity_provision.go

```diff
@@ -0,0 +1,92 @@
+package upgrade_kyma
+
+import (
+	"fmt"
+	"time"
+
+	"github.com/kyma-project/control-plane/components/kyma-environment-broker/internal/process/provisioning"
+
+	"github.com/google/uuid"
+	"github.com/kyma-project/control-plane/components/kyma-environment-broker/internal"
+	"github.com/kyma-project/control-plane/components/kyma-environment-broker/internal/process"
+	"github.com/kyma-project/control-plane/components/kyma-environment-broker/internal/servicemanager"
+	"github.com/kyma-project/control-plane/components/kyma-environment-broker/internal/storage"
+	"github.com/sirupsen/logrus"
+)
+
+type ConnectivityUpgradeProvisionStep struct {
+	operationManager *process.UpgradeKymaOperationManager
+}
+
+func NewConnectivityUpgradeProvisionStep(os storage.Operations) *ConnectivityUpgradeProvisionStep {
+	return &ConnectivityUpgradeProvisionStep{
+		operationManager: process.NewUpgradeKymaOperationManager(os),
+	}
+}
+
+var _ Step = (*ConnectivityUpgradeProvisionStep)(nil)
+
+func (s *ConnectivityUpgradeProvisionStep) Name() string {
+	return "CONNECTIVITY_UpgradeProvision"
```

Comment:
```suggestion
	return "Connectivity_UpgradeProvision"
```

Reply:
Apart from that I find some inconsistency in upgrade steps naming convention...

Reply:
Do you mean UpgradeProvision vs Upgrade? I left UpgradeProvision in upgrade steps.